lib/auth/permissions.go

```diff
@@ -298,6 +298,7 @@ func GetCheckerForBuiltinRole(clusterName string, clusterConfig services.Cluster
 			services.RoleSpecV3{
 				Allow: services.RoleConditions{
 					Namespaces: []string{services.Wildcard},
+					AppLabels:  services.Labels{services.Wildcard: []string{services.Wildcard}},
```

Comment:
Hmm, I don't think RoleNode (which represents an SSH node) should have AppLabels set?

Reply:
Good point, I had thought there was a test that wanted this but apparently not, I've removed it now.